pkgs/applications/editors/music/tuxguitar/default.nix

```diff
@@ -0,0 +1,40 @@
+{ system, stdenv, fetchurl, swt, jdk, makeWrapper, alsaLib }:
+
+let metadata = if system == "i686-linux"
+               then { arch = "x86"; sha256 = "1qmb51k0538pn7gv4nsvhfv33xik4l4af0qmpllkzrikmj8wvzlb"; }
+               else if system == "x86_64-linux"
+                    then { arch = "x86_64"; sha256 = "12af47jhlrh9aq5b3d13l7cdhlndgnfpy61gz002hajbq7i00ixh"; }
+                    else { };
+in stdenv.mkDerivation rec {
+  version = "1.2";
+  name = "tuxguitar-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/tuxguitar/${name}-linux-${metadata.arch}.tar.gz";
+    sha256 = metadata.sha256;
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
After a good night of sleep, I realized the build did not depend on swt, jdk and alsaLib.
Fixed it.